### ISSUE TITLE
[redis-operator] bug watch_namespace template

### ISSUE THREAD

**Issue by vramperez** [OT-CONTAINER-KIT/helm-charts#204]
**Does this issue reproduce with the latest release?**
yes

**What operating system and processor architecture are you using (`kubectl version`)?**
arch: x86_64
OS: Fedora 39

<details><summary><code>kubectl version</code> Output</summary><br><pre>
$ kubectl version
Client Version: v1.30.0
Kustomize Version: v5.0.4-0.20230601165947-6ce0bf390ce3
Server Version: v1.29.4-eks-036c24b
</pre></details>

**What did you do?**
`helm template redis-operator charts/redis-operator --set redisOperator.watch_namespace=test`

**What did you expect to see?**
```yaml
...
        env:
        - name: ENABLE_WEBHOOKS
          value: "false"
        - name: WATCH_NAMESPACE
          value: test
...
```

**What did you see instead?**
`watch_namespace` property is ignored:

```yaml
...
        env:
        - name: ENABLE_WEBHOOKS
          value: "false"
...
```

**Comment by Mortega5** [comment]
fixed in [Rediss operator PR #990](https://github.com/OT-CONTAINER-KIT/redis-operator/pull/990)

**Comment by vramperez** [comment]
Fixed in https://github.com/OT-CONTAINER-KIT/redis-operator/commit/052d2e5e0d21b88ba6b9d5d4e9b31e51ae182a6d
